### DISCUSSION TITLE
use of Rust based stretch layout engine instead of Yoga Layout engine

### DISCUSSION THREAD

**Discussion by maurya-ankit** [vercel/satori#428] [Ideas]
https://medium.com/visly/stretch-a-flexbox-implementation-in-rust-60762b5a3331

with the current Satori configuration, when using content-centering properties in JSX, I can see latencies getting higher.

can this be lowered down to a certain extent by using Rust based Stretch Layout Engine?

**Comment by jeetiss** [comment]
why do you want to use `stretch`? it is unmaintained and has a lot of problems with leaf nodes and measure functions.
There is [taffy](https://github.com/DioxusLabs/taffy). It looks promising and supports grid layout, but doesn't have wasm bindings at this moment
